lib/graphql/define/defined_object_proxy.rb

```diff
@@ -11,6 +11,15 @@ def types
         GraphQL::Define::TypeDefiner.instance
       end
 
+      def use(plugin_class, *args, **kwargs)
+        # https://bugs.ruby-lang.org/issues/10708
+        if kwargs == {}
+          plugin_class.use(self, *args)
+        else
+          plugin_class.use(self, *args, **kwargs)
```

Comment:
We don't expect this to be a `Class`, right? Only something that responds to `use`?